webhooks/pod_webhook.go

```diff
@@ -74,10 +75,16 @@ func (m *PodMutator) Handle(ctx context.Context, req admission.Request) admissio
 		return admission.Denied("static or orphaned pods cannot be mutated")
 	}
 
-	if code, err := m.handleRPCEvaluation(ctx, req, annotations, pod); err != nil {
-		if code == 0 {
-			return admission.Denied(err.Error())
-		} else {
+	if shouldUseRPC(annotations) {
+		if code, err := m.handleRPCEvaluation(ctx, req, annotations, pod); err != nil {
+			if code == 0 {
```

Comment:
to make if easier to understand the semantics of the `code` returned by this function, should we rather make it return `http.StatusForbidden` in this case? since this is also what will be set in `admission.Denied()`